rasa/core/policies/flow_policy.py

```diff
@@ -194,7 +270,7 @@ def _create_prediction_result(
         if action_name:
             result[domain.index_for_action(action_name)] = score
         return self._prediction(
-            result, optional_events=events, action_metadata=action_metadata
+            result, events=events, action_metadata=action_metadata
```

Comment:
this is a bit dangerous as these events will always be applied, but esp. the flow ones should only be used if this policy wins. otherwise we might be applying events even though another policy won